sda-commons-server-morphia/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
   compile project(':sda-commons-server-dropwizard')
+  compile project(':sda-commons-starter')
```

Comment:
Could we depend on `sda-commons-shared-certificates`? I think, it is not necessary to include tho whole starter bundle.

Reply:
done